### ISSUE TITLE
Deploy fixes for Flow types to NPM

### ISSUE THREAD

**Issue by sebpearce** [developit/mitt#59]
I noticed that #47 fixes some Flow types issues, but that the `package.json` is older than that PR. It's causing type errors for us via our `svg-baker-runtime` package which uses Mitt as a dependency. We're ignoring Mitt in our `.flowconfig` temporarily, but you might want to make sure it's been deployed.

**Comment by egormerkushev** [comment]
I have the same problem!
@sebpearce Please, can you share your ignore line for the mitt?

**Comment by ndbroadbent** [comment]
+1, I'm getting this error when I run flow:

```
Error: node_modules/mitt/src/index.js:54
 54: 			(all['*'] || []).map((handler) => { handler(type, evt); });
     			                                                  ^^^ unused function argument
    4: type EventHandler = (event?: any) => void;
                           ^^^^^^^^^^^^^^^^^^^^^ function type expects no more than 1 argument
```

**Comment by sebpearce** [comment]
@egormerkushev So sorry, I somehow missed the notification about this!

In case anyone else is interested, the ignore line was:
```
[ignore]
.*node_modules/mitt/.*
```

**Comment by charlike** [comment]
yep, sadly i can't help here, cuz dont have npm rights.

We definitely need new versions, because even the build for the latest published versions is failing. So new versions will get that https://github.com/developit/mitt/commit/9953570dac00c2d1a9d90fc845aa8a39037729ca published too.

**Comment by developit** [comment]
Hiya - sorry about that, I was on a bit of a break from OSS but I'm back :)  I'll publish.

**Comment by developit** [comment]
I just released 1.1.3, it should be fixed.
